app/src/pages/playground/schemas.ts

```diff
@@ -219,3 +257,23 @@ export const llmToolSchema = z
   .optional();
 
 export type LlmToolSchema = z.infer<typeof llmToolSchema>;
+
+export const openAIResponseFormatSchema = z.lazy(() =>
+  z.object({
+    type: z.literal("json_schema"),
+    json_schema: z.object({
+      name: z.string().describe("The name of the schema"),
+      schema: jsonLiteralSchema,
+      strict: z.literal(true).describe("The schema must be strict"),
```

Comment:
nit:

```suggestion
      strict: z.literal(true).describe("Whether or not the schema must be strict"),
```

Reply:
This is not user configurable, openai requires it to be true

Reply:
ohhh interesting haha i did not read the z.literal